For the record: seems like that if I change the version of `pytensor` and  `pymc` to `pytensor>=2.28.0`
`pymc>=5.21.0` in the environment files, `devito` produces a strange compilation error in `plot_twoway` in my local ARM environment.

```
self = JITCompiler[CustomCompiler], ext_file = '/var/folders/k2/5r6ztqx10jscz2r52gj5ry2c0000gn/T/devito-jitcache-uid501/790839b6ddf8bcf29121cdd8508fbe03d7782ca4.dylib'
source_files = ['/var/folders/k2/5r6ztqx10jscz2r52gj5ry2c0000gn/T/devito-jitcache-uid501/790839b6ddf8bcf29121cdd8508fbe03d7782ca4.c'], debug = False

    def build_extension(self, ext_file, source_files, debug=False):
        cc_cmdline = (
                self._cmdline(source_files, False)
                + ["-o", ext_file]
                )
    
        from pytools.prefork import call
        if debug:
            print(" ".join(cc_cmdline))
    
        result = call(cc_cmdline)
    
        if result != 0:
            import sys
            print("FAILED compiler invocation: {}".format(" ".join(cc_cmdline)),
                  file=sys.stderr)
>           raise CompileError("module compilation failed")
E           codepy.CompileError: module compilation failed

/opt/miniconda3/envs/pylops/lib/python3.13/site-packages/codepy/toolchain.py:211: CompileError
```

For now, I revert back the environment files to `pymc>=5` and `pytensor` as this seems to work fine.

--- a/environment-dev-arm.yml
+++ b/environment-dev-arm.yml
@@ -11,8 +11,8 @@ dependencies:
   - pyfftw
   - pywavelets
   - sympy
-  - pytensor>=2.28.0
-  - pymc>=5.21.0
+  - pymc>=5
+  - pytensor
   - matplotlib
   - ipython
   - pytest

--- a/environment-dev.yml
+++ b/environment-dev.yml
@@ -11,8 +11,8 @@ dependencies:
   - pyfftw
   - pywavelets
   - sympy
-  - pytensor>=2.28.0
-  - pymc>=5.21.0
+  - pymc>=5
+  - pytensor
   - matplotlib
   - ipython
   - pytest

